notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>Suggested rewording of how the data were collected: "Examining the full amount of data available, we can see that the target of interest was observed in two TESS Sectors back-to-back (Sectors 18 and 19) and then again in Sectors 24 and 25, with a gap of approximately 100 days between the end of Sector 19 and the beginning of Sector 24."</p><p> </p><p>Spelling correction: "Data <strong>Gaps</strong> and Quality Flags"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>